site/content/docs/5.1/utilities/api.md

```diff
@@ -136,12 +137,15 @@ Output:
 
 ### CSS variable utilities
 
-Set the `css-var` boolean option to `true` and the API will generate local CSS variables for the given selector instead of the usual `property: value` rules. Consider our `.text-opacity-*` utilities:
+Set the `css-var` boolean option to `true` and the API will generate local CSS variables for the given selector instead of the usual `property: value` rules. Add an optional `css-variable-name` have a different variable name than the class name.
+
+Consider our `.text-opacity-*` utilities:
 
 ```scss
 $utilities: (
   "text-opacity": (
     css-var: true,
+    css-variable-name: text-opacity,
     class: text-opacity,
```

Comment:
Is this a good example? If I understand the changes in utilities API, result would be the same if we dropped `css-variable-name`, wouldn't it?